### ISSUE TITLE
1.4.4 crashes on opening inventory: java.lang.IllegalArgumentException: Duplicate delegates

### ISSUE THREAD

**Issue by mmodrow** [IrisShaders/Iris#1777]
### Minecraft Version

1.19.2

### Iris Version

iris-mc1.19.2-1.4.4.jar

### Sodium Version

sodium-fabric-mc1.19.2-0.4.4+build.18.jar

### Operating System

Windows 10

### What is your GPU?

NVIDIA GeForce RTX 3080

### Java Version

17.0.3

### Reproduction Steps

1. Open Inventory in Survival Mode
2. Game Crashes

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 2022-12-07 20:55:10
Description: Rendering entity in world

java.lang.IllegalArgumentException: Duplicate delegates
	at net.minecraft.class_4720$class_4589.<init>(class_4720.java:29)
	at net.minecraft.class_4720.method_24037(class_4720.java:15)
	at net.minecraft.class_918.method_27952(class_918.java:178)
	at net.minecraft.class_970.method_23192(class_970.java:105)
	at net.minecraft.class_970.method_4169(class_970.java:70)
	at net.minecraft.class_970.method_17157(class_970.java:36)
	at net.minecraft.class_970.method_4199(class_970.java:22)
	at net.minecraft.class_922.method_4054(class_922.java:145)
	at net.minecraft.class_1007.method_4215(class_1007.java:63)
	at net.minecraft.class_1007.method_3936(class_1007.java:41)
	at net.minecraft.class_898.method_3954(class_898.java:141)
	at net.minecraft.class_490.method_29977(class_490.java:152)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(RenderSystem.java:1171)
	at net.minecraft.class_490.method_2486(class_490.java:152)
	at net.minecraft.class_490.method_2389(class_490.java:110)
	at net.minecraft.class_465.method_25394(class_465.java:100)
	at net.minecraft.class_485.method_25394(class_485.java:27)
	at net.minecraft.class_490.method_25394(class_490.java:90)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_4720$class_4589.<init>(class_4720.java:29)
	at net.minecraft.class_4720.method_24037(class_4720.java:15)
	at net.minecraft.class_918.method_27952(class_918.java:178)
	at net.minecraft.class_970.method_23192(class_970.java:105)
	at net.minecraft.class_970.method_4169(class_970.java:70)
	at net.minecraft.class_970.method_17157(class_970.java:36)
	at net.minecraft.class_970.method_4199(class_970.java:22)
	at net.minecraft.class_922.method_4054(class_922.java:145)
	at net.minecraft.class_1007.method_4215(class_1007.java:63)
	at net.minecraft.class_1007.method_3936(class_1007.java:41)
	at net.minecraft.class_898.method_3954(class_898.java:141)
	at net.minecraft.class_490.method_29977(class_490.java:152)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(RenderSystem.java:1171)
	at net.minecraft.class_490.method_2486(class_490.java:152)
	at net.minecraft.class_490.method_2389(class_490.java:110)
	at net.minecraft.class_465.method_25394(class_465.java:100)
	at net.minecraft.class_485.method_25394(class_485.java:27)
	at net.minecraft.class_490.method_25394(class_490.java:90)

-- Entity being rendered --
Details:
	Entity Type: minecraft:player (net.minecraft.class_746)
	Entity ID: 7111
	Entity Name: Typ_im_Dunkelln
	Entity's Exact location: 1830.87, 78.00, 801.44
	Entity's Block location: World: (1830,78,801), Section: (at 6,14,1 in 114,4,50; chunk contains blocks 1824,-64,800 to 1839,319,815), Region: (3,1; contains chunks 96,32 to 127,63, blocks 1536,-64,512 to 2047,319,1023)
	Entity's Momentum: 0.00, -0.08, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at net.minecraft.class_898.method_3954(class_898.java:141)
	at net.minecraft.class_490.method_29977(class_490.java:152)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(RenderSystem.java:1171)
	at net.minecraft.class_490.method_2486(class_490.java:152)
	at net.minecraft.class_490.method_2389(class_490.java:110)
	at net.minecraft.class_465.method_25394(class_465.java:100)
	at net.minecraft.class_485.method_25394(class_485.java:27)
	at net.minecraft.class_490.method_25394(class_490.java:90)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Renderer details --
Details:
	Assigned renderer: net.minecraft.class_1007@6ae57089
	Location: 0.00,0.00,0.00 - World: (0,0,0), Section: (at 0,0,0 in 0,0,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Rotation: 0.0
	Delta: 1.0
Stacktrace:
	at net.minecraft.class_898.method_3954(class_898.java:141)
	at net.minecraft.class_490.method_29977(class_490.java:152)
	at com.mojang.blaze3d.systems.RenderSystem.runAsFancy(RenderSystem.java:1171)
	at net.minecraft.class_490.method_2486(class_490.java:152)
	at net.minecraft.class_490.method_2389(class_490.java:110)
	at net.minecraft.class_465.method_25394(class_465.java:100)
	at net.minecraft.class_485.method_25394(class_485.java:27)
	at net.minecraft.class_490.method_25394(class_490.java:90)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Screen render details --
Details:
	Screen name: net.minecraft.class_490
	Mouse location: Scaled: (320, 180). Absolute: (960.000000, 540.000000)
	Screen size: Scaled: (640, 360). Absolute: (1920, 1080). Scale factor of 3.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 1 total; [class_746['Typ_im_Dunkelln'/7111, l='ClientLevel', x=1830.87, y=78.00, z=801.44]]
	Chunk stats: 4096, 1024 F: 0 L: 0 U: 650
	Level dimension: minecraft:overworld
	Level spawn location: World: (-21,188,-60), Section: (at 11,12,4 in -2,11,-4; chunk contains blocks -32,-64,-64 to -17,319,-49), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,-64,-512 to -1,319,-1)
	Level time: 310433982 game time, 329961517 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2341)
	at net.minecraft.class_310.method_1514(class_310.java:787)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, automated_crafting/powered_icon, automated_crafting/powered_block, FreshAnimations_v1.7_S8.zip, VT_20220928_1.19_utility.zip, VT_20220928_1.19_utility_Blockdata.zip, VT_20221124_1.19_main.zip

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1338431552 bytes (1276 MiB) / 2818572288 bytes (2688 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 5800X 8-Core Processor             
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: unknown
	Frequency (GHz): 3.79
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 3080
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2216
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2737
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 41731.28
	Virtual memory used (MB): 36495.94
	Swap memory total (MB): 9060.64
	Swap memory used (MB): 1459.81
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx6G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		animatica: Animatica 0.5+1.19
		automated_crafting: Automated Crafting 1.4.7
			crowdin-translate: CrowdinTranslate 1.4+1.19
			libgui: LibGui 6.0.0+1.19
				jankson: Jankson 4.1.1+j1.2.1
					blue_endless_jankson: jankson 1.2.1
				libninepatch: LibNinePatch 1.1.0
		betterf3: BetterF3 1.3.2
		bobby: Bobby 4.0.1
			com_typesafe_config: config 1.4.2
			io_leangen_geantyref_geantyref: geantyref 1.3.13
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		carpet: Carpet Mod 1.4.84
		cem: Custom Entity Models 0.7.1_S7_1.19
			completeconfig-base: completeconfig-base 2.0.0
			completeconfig-gui-cloth: completeconfig-gui-cloth 2.0.0
		cloth-config: Cloth Config v8 8.2.88
			cloth-basic-math: cloth-basic-math 0.6.1
		entityculling: EntityCulling-Fabric 1.5.2-mc1.19
			com_logisticscraft_occlusionculling: occlusionculling 0.0.6-SNAPSHOT
		fabric-api: Fabric API 0.67.1+1.19.2
			fabric-api-base: Fabric API Base 0.4.13+0d0f210290
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.12+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.20+0d0f210290
			fabric-block-api-v1: Fabric Block API (v1) 1.0.1+12bfe4ea90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.22+c6af733c90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.3+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.13+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.9+93d8cb8290
			fabric-commands-v0: Fabric Commands (v0) 0.2.30+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.37+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.3+6064db2a90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.3+93d8cb8290
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.7+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.3.1+ad31814d90
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.33+0d0f210290
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.2+9244241690
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.30+c6af733c90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.31+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.23+c6af733c90
			fabric-item-api-v1: Fabric Item API (v1) 1.6.1+fa140d5990
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.35+9244241690
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.23+aaaf9d3390
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.21+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.2+9244241690
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.9+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.12+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.5+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.20+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.19+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.7+9244241690
			fabric-networking-v0: Fabric Networking (v0) 0.3.24+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.1.2+06e0e25190
			fabric-particles-v1: Fabric Particles (v1) 1.0.12+79adfe0a90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.28+9244241690
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.14+12bfe4ea90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.16+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.22+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.17+0d0f210290
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.9+c6af733c90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.25+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.1+73145abb90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.1+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.8.1+0d0f210290
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.29+0d0f210290
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.3+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.1+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.22+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.3+9d081d6490
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.2+42d99c3290
		fabricloader: Fabric Loader 0.14.9
		inventorysorter: Inventory Sorter 1.8.10-1.19
			kyrptconfig: Kyrpt Config 1.4.14-1.19
		iris: Iris 1.4.4
			io_github_douira_glsl-transformer: glsl-transformer 1.0.1
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.10.1
		java: OpenJDK 64-Bit Server VM 17
		litematica: Litematica 0.12.5
		lithium: Lithium 0.10.2
		malilib: MaLiLib 0.13.0
		minecraft: Minecraft 1.19.2
		minihud: MiniHUD 0.23.3
		modmenu: Mod Menu 4.1.1
		quickcraft: Quickcraft 1.2.5
		reeses-sodium-options: Reese's Sodium Options 1.4.7+mc1.19.2-build.59
		sodium: Sodium 0.4.4+build.18
			org_joml_joml: joml 1.10.4
		sodium-extra: Sodium Extra 0.4.13+mc1.19.2-build.76
			caffeineconfig: CaffeineConfig 1.0.0+1.17
		sound_physics_remastered: Sound Physics Remastered 1.19.2-1.0.15
		starlight: Starlight 1.1.1+fabric.ae22326
		syncmatica: Syncmatica 1.18.2-0.3.4
		tweakeroo: Tweakeroo 0.14.3
		voicechat: Simple Voice Chat 1.19.2-2.3.16
		worldedit: WorldEdit 7.2.12+6240-87f4ae1
		worldeditcui: WorldEditCUI 1.19.2+01
		xaerominimap: Xaero's Minimap 22.16.2
		xaeroworldmap: Xaero's World Map 1.28.3
	Loaded Shaderpack: ComplementaryReimagined_r1.2.1.zip
		Profile: Custom (+7 options changed by user)
	NEC status: No NEC detected
	Launched Version: fabric-loader-0.14.9-1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 3080/PCIe/SSE2 GL version 3.2.0 NVIDIA 527.37, NVIDIA Corporation
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, Fabric Mods, automated_crafting/powered_icon (incompatible), automated_crafting/powered_block (incompatible), file/FreshAnimations_v1.7_S8.zip, file/VT_20220928_1.19_utility.zip, file/VT_20220928_1.19_utility_Blockdata.zip, file/VT_20221124_1.19_main.zip
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 5800X 8-Core Processor
```

### Additional context

Downgrading to 1.4.3 instantly removed the error. This tells me, it's something in the changes between 1.4.3 and 1.4.4.

**Comment by IMS212** [comment]
Problem has been identified, and a fix will be coming shortly.

**Comment by IMS212** [comment]
Fixed in Iris 1.4.5.
